omnibus/config/projects/agent.rb

```diff
@@ -321,6 +324,16 @@
     end
   }
 
+  raise_if_fips_symbol_not_found = Proc.new { |symbols|
+      count = symbols.scan("github.com/microsoft/go-crypto-winnative").count()
+      if count == 0
+        puts "Checking for FIPS symbols: '#{symbols}'"
```

Comment:
Should we move this `puts` above? Looks like we only print this line if we do not find the symbol

Reply:
We only want to log this on failure because `go tool nm` is so huge that GitLab reaches its log limit :(

Reply:
```suggestion
        log.debug(log_key) { "Checking for FIPS symbols: '#{symbols}'" }
```
in all fairness I'm not 100% sure this will work out of the box and it might need some small adjustments, but it would be better to abide by the log level and not dump some logs unconditionally in the output

Reply:
> We only want to log this on failure because go tool nm is so huge that GitLab reaches its log limit :(

If that is the case it looks like the log message might need a rewording. Or even removing it as the exception already includes information